devices.js

```diff
@@ -1204,9 +1204,9 @@ const devices = [
         description: 'Smart+ switch mini',
         supports: 'on/off, brightness',
         fromZigbee: [
-            fz.genOnOff_cmdOn, fz.genOnOff_cmdOff, fz.AC0251100NJ_long_middle,
-            fz.cmd_stop, fz.cmd_move, fz.cmd_move_with_onoff,
-            fz.cmd_move_to_level_with_onoff, fz.generic_batteryvoltage_3000_2500,
+            fz.AC0251100NJ_on, fz.AC0251100NJ_off, fz.AC0251100NJ_on_hold, fz.AC0251100NJ_off_hold,
+            fz.AC0251100NJ_release, fz.AC0251100NJ_circle, fz.AC0251100NJ_circle_release,
+            fz.generic_batteryvoltage_3000_2500, fz.cmd_stop,
```

Comment:
I think the fz.cmd_stop should be removed and replaced with a different once (as you did with all the others).

Reply:
with a different once ??

Reply:
https://github.com/Koenkk/zigbee-shepherd-converters/blob/master/converters/fromZigbee.js#L1471, it use the `ictcg1` method, for this, it also should use `cmd_move` etc, but these are removed in this commit. Meaning that this once should also be removed/replaced.

Reply:
ahh .. yes i think we can remove this fz.cmd_stop